vision/google/cloud/proto/vision/v1/text_annotation_pb2.py

```diff
@@ -558,13 +558,14 @@
   DESCRIPTOR = _TEXTANNOTATION,
   __module__ = 'google.cloud.proto.vision.v1.text_annotation_pb2'
   ,
-  __doc__ = """TextAnnotation contains a structured representation of OCR extracted
-  text. The hierarchy of an OCR extracted text structure is like this:
-  TextAnnotation -> Page -> Block -> Paragraph -> Word -> Symbol Each
+  __doc__ = """``TextAnnotation`` contains a structured representation of OCR extracted text.
```

Comment:
I would be, could you point me to the unit test module / file that needs updating?